scripts/export_sqlite_to_workspace.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
```

Comment:
I'd prefer to use the Python logger capabilities, so you have a nice way to control verbosity